codice/ddf
# [2.13.x] DDF-04415 Solr stop key for windows

#### What does this PR do?
Solr stop key was not properly set on Windows.
This PR addresses that.

#### Who is reviewing it? 
@austinsteffes @tyler30clemens @garrettfreibott @brjeter 
@codice/solr 

#### Ask 2 committers to review/merge the PR and tag them here.

@brjeter
@mcalcote 
@vinamartin

#### How should this be tested?
From the terminal, test starting and stopping Solr.

Test 1 
- Precondition: Solr is NOT running
- Precondition: Terminal in the DDF bin directory
- Run `ddfsolr.bat start`
- Expected result 1: Solr is running
- Expected result 2: A file named STOPKEY is in the bin directory and it is 8 bytes long
- Run `ddfsolr.bat stop`
- Expected result 1: Solr is NOT running
- Expected result 2: The STOPKEY has been removed

Test 2
- Precondition: Solr and DDF are NOT running
- Precondition: Terminal in the DDF bin directory
- Run `ddf.bat start`
- Expected result 1: The DDF starts and the DDF log shows a successful connection to Solr
- Expected result 2: A file named STOPKEY is in the bin directory and it is 8 bytes long
- Shutdown DDF
- Expected result 1: Solr and DDF are NOT running
- Expected result 2: The STOPKEY has been removed

For GH Issues:
Fixes: #4415 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an inline code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.
